EIPS/eip-restricted_chain_context_txs.md

```diff
@@ -0,0 +1,180 @@
+
+##### Encoding
+
+Encoding should follow the same rules as the optional `data` value.
+
+#### `eligibleMinerList`
+
+- `ANNOTATION_PREFIX` `2`.
+- `ELIGIBLE_MINER_LIST` `[address...]`. A list of addresses.
+- `MAX_ELEMENTS` `10`. The maximum number of addresses that can be provided.
+
+Constrains the validity of a transaction by a chain context subclass referencing the miner address of the block including
+the transaction.
+The transaction is only valid when included in a block having an `etherbase` contained in annotated list of addresses.
+
+##### Encoding
+
+Encoding should follow the same rules as the optional `accessList` value.
+
+#### `expiry`
+
+- `ANNOTATION_PREFIX` `4`. 
+
+Constrains the validity of a transaction by a chain context subclass referencing the timestamp of the block including the transaction (in seconds-since Unix epoch).
+The transaction is only valid when included in a block having a `timestamp` less than the value annotated.
+
+##### Encoding
+
+Encoding should follow the same rules as other transaction integer values, eg. `nonce`, `gasPrice`, `gasLimit`, etc.
+
+### Subclass Combination
+
+Since subclasses use octal values for `ANNOTATION_PREFIX`, they can be distinguishably combined as long as
+we assume annotation cardinality (ie ordering).
+
+For example:
+
+- `ANNOTATION_PREFIX` `1` signals `segmentId` exclusively. 
+- `ANNOTATION_PREFIX` `2` signals `eligibleMinerList` exclusively. 
+- `ANNOTATION_PREFIX` `3` combines `segmentId` and `eligibleMinerList`.
+- `ANNOTATION_PREFIX` `4` signals `expiry` exclusively.
+- `ANNOTATION_PREFIX` `5` combines `segmentId` and `expiry`. 
+- `ANNOTATION_PREFIX` `6` combines `eligibleMinerList` and `expiry`. 
+- `ANNOTATION_PREFIX` `7` combines `segmentId` and `eligibleMinerList` and `expiry`. 
+
+
+Annotation values are provided in an array.
+
+For example:
+
+- `chainContext` `1[c0e1e4e78b1ec3]` Transaction can only be included in a block having a canonical ancestor block numbered `15_000_000` and with a hash prefixed with the bytes `e78b1ec3`.
+- `chainContext` `2[[Df7D7e053933b5cC24372f878c90E62dADAD5d42]]` Transaction can only be included in a block naming `Df7D7e053933b5cC24372f878c90E62dADAD5d42` as the `etherbase` beneficiary.
+- `chainContext` `3[1619008030]` Transaction can only be included in a block having a timestamp with value less than the time signaled by the Unix-epoch value `1619008030` (generally, the transaction expires after Wed Apr 21 07:27:10 CDT 2021).
+
+
+## Rationale
+
+__EIP2930: Optional Access List Type Transactions__ is used as an assumed "base" transaction type for this
+proposal. However, this is NOT a conceptual dependency; the included `accessList` portion of this proposal
+can readily be removed. Standing on the shoulders of EIP2930 is only intended to support and further
+the adoption of next-generation transactions.
+
+
+## Reference
+
+## Copyright
```

Comment:
```suggestiotn
## Backwards Compatibility
TBD

## Test Cases
TBD

## Security Considerations
TBD

## Copyright
```
For the test cases, I recommend moving the examples you have in the specification section into the test cases section.